### ISSUE TITLE
Metadata values for uploads are replaced incorrectly

### ISSUE THREAD

**Issue by onnimonni** [devgeniem/wp-sanitize-accented-uploads#3]
`$ wp sanitize all` will change the metadata of your files incorrectly in some cases.

Here's example of metadata from my site:

```
wp eval "var_dump(wp_get_attachment_metadata(9));"
array(5) {
  ["width"]=>
  int(418)
  ["height"]=>
  int(280)
  ["file"]=>
  string(17) "2014/11/test1.jpg"
  ["sizes"]=>
  array(5) {
    ["thumbnail"]=>
    array(4) {
      ["file"]=>
      string(17) "test1-150x150.jpg"
      ["width"]=>
      int(150)
      ["height"]=>
      int(150)
      ["mime-type"]=>
      string(10) "image/jpeg"
    }
    ["medium"]=>
    array(4) {
      ["file"]=>
      string(17) "test1-280x160.jpg"
      ["width"]=>
      int(280)
      ["height"]=>
      int(160)
      ["mime-type"]=>
      string(10) "image/jpeg"
    }
    ["small"]=>
    array(4) {
      ["file"]=>
      string(16) "test1-107x75.jpg"
      ["width"]=>
      int(107)
      ["height"]=>
      int(75)
      ["mime-type"]=>
      string(10) "image/jpeg"
    }
    ["map-img"]=>
    array(4) {
      ["file"]=>
      string(17) "test1-152x108.jpg"
      ["width"]=>
      int(152)
      ["height"]=>
      int(108)
      ["mime-type"]=>
      string(10) "image/jpeg"
    }
    ["custom-size"]=>
    array(4) {
      ["file"]=>
      string(17) "test1-418x200.jpg"
      ["width"]=>
      int(418)
      ["height"]=>
      int(200)
      ["mime-type"]=>
      string(10) "image/jpeg"
    }
  }
  ["image_meta"]=>
  array(11) {
    ["aperture"]=>
    int(0)
    ["credit"]=>
    string(0) ""
    ["camera"]=>
    string(0) ""
    ["caption"]=>
    string(0) ""
    ["created_timestamp"]=>
    int(0)
    ["copyright"]=>
    string(0) ""
    ["focal_length"]=>
    int(0)
    ["iso"]=>
    int(0)
    ["shutter_speed"]=>
    int(0)
    ["title"]=>
    string(0) ""
    ["orientation"]=>
    int(1)
  }
}
```

**Comment by onnimonni** [comment]
Fixed in 84fc87eece86dd0427d12c5593b5f4c0400dcd32

**Comment by rhurling** [comment]
Thank you :)
